### ISSUE TITLE
List of locales

### ISSUE THREAD

**Issue by vizo** [catberry/catberry-l10n#17]
How can i get in browser list of all locales available (loaded)?

On server i can do by Object.keys(this._l10n._loader._fileCache), but on client i can't?

**Comment by pragmadash** [comment]
Yeah, agree, there is no way to get a list of all supported locales currently. A good feature request, though.
